extensions/wikia/ThemeDesigner/UploadWordmarkFromFile.class.php

```diff
@@ -1,59 +1,60 @@
 <?php
+
 /*
  * @author Federico "Lox" Lucignano
  */
 
 class UploadWordmarkFromFile extends UploadFromFile {
 	const FILEDIMENSIONS_ERROR = 30;
 	const FILETYPE_ERROR = 31;
-	
-	public function verifyUpload(){
+
+	public function verifyUpload() {
 		$this->getTitle(); //will fill in final destination and extension
 		$details = parent::verifyUpload();
-		
-		if ( $details[ 'status' ] == self::OK ){ 
+
+		if ( $details['status'] == self::OK ) {
 			// check file type (just by extension)
-			
-			if ( $this->checkFileExtension( $this->mFinalExtension, array( 'png' ) ) ) {
+
+			if ( $this->checkFileExtension( $this->mFinalExtension, [ 'png' ] ) ) {
 				$imageSize = getimagesize( $this->getTempPath() );
 
-				if($imageSize[0] > 250 || $imageSize[1] > 65) {
-					$details[ 'status' ] = self::FILEDIMENSIONS_ERROR;
+				if ( $imageSize[0] > 250 || $imageSize[1] > 65 ) {
+					$details['status'] = self::FILEDIMENSIONS_ERROR;
 				}
 			} else {
-				$details[ 'status' ] = self::FILETYPE_ERROR;
+				$details['status'] = self::FILETYPE_ERROR;
 			}
 		}
 
 		return $details;
 	}
-	
-	
+
+
 	public function performUpload() {
 		global $wgUser;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)